app/controllers/remotes_controller.rb

```diff
@@ -0,0 +1,61 @@
+class RemotesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_remote, only: [:show, :edit, :update]
+
+  # GET /remotes
+  def index
+    @remotes = Remote.all
+  end
+
+  # GET /remotes/1
+  def show
+  end
+
+  # GET /remotes/new
+  def new
+    @remote = Remote.new
+  end
+
+  # GET /remotes/1/edit
+  def edit
+  end
+
+  # PATCH/PUT /ext_apps/1
+  def update
+    @remote = Remote.find(params[:id])
+
+    if @remote.update(remote_params)
+      redirect_to @remote
+    else
+      render 'edit'
+    end
+  end
+
+  # POST /remotes
+  def create
+    @remote = Remote.create(remote_params)
```

Comment:
why not use similar pattern with other controllers?
`#new`
then `if @remote.save` below

Reply:
Noted

Reply:
Done. https://github.com/pathfinder-cm/pathfinder-mono/pull/26/commits/a7e9687e061cf8a7b413431583b6af262206507a